posix-configs/rpi2/mainapp.config

```diff
@@ -3,11 +3,13 @@ param set SYS_AUTOSTART 4001
 param set MAV_BROADCAST 1
 sleep 1
 param set MAV_TYPE 2
-df_mpu9250_wrapper start
-df_hmc5883_wrapper start
+df_lsm9ds1_wrapper start -R 4
+#df_mpu9250_wrapper start -R 10
```

Comment:
Well sometimes it just fails and keeps saying "FIFO overflow" which is unfortunate.